Components/TextTitle.js

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import '../css/Components/TextTitle';
+
+const TextTitle = ({ children, alternate, dark }) => {
+  const className = classNames({
+    TextTitle: true,
+    'TextTitle--alternate': alternate,
+    'TextTitle--dark': dark,
```

Comment:
As far as I can see, this modifier is only used in one specific situation. Why didn't you added the text style to the `Feedback` component directly?